src/components/UserProfilePopUp/UserProfileToggler.js

```diff
@@ -2,13 +2,11 @@ import React, { useState } from "react";
 import Icon from "@material-ui/core/Icon";
 import { withStyles } from "@material-ui/styles";
 import clsx from "clsx";
-import { connect } from "react-redux";
 
 import { useStyles } from "./styles";
 import UserProfilePopUp from "./";
-import UserBlockies from "../common/UserBlockies";
```

Comment:
If we are not going to use Blockies anymore then I think we should also delete the associated file and package dependencies.

Reply:
[removed](https://github.com/singnet/snet-dapp/pull/499/commits/af01a202937664c2a22ce355861db8efd4c41334)